services/src/main/java/org/keycloak/broker/oidc/mappers/AbstractClaimMapper.java

```diff
@@ -122,6 +123,13 @@ public boolean valueEquals(String desiredValue, Object value) {
             for (Object val : list) {
             	if (valueEquals(desiredValue, val)) return true;
             }
+        } else if (value instanceof JsonNode) {
+            try {
+                final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
You should consider using `org.keycloak.util.JsonSerialization` instead of creating a new `ObjectMapper` instance.